packages/forms/src/UIForm/fieldsets/Columns/Columns.component.js

```diff
@@ -10,9 +10,12 @@ export default function Columns(props) {
 
 	return (
 		<div className={classNames('tf-columns', theme['tf-columns'])}>
-			{schema.items.map(
-				(colSchema, index) => (<Widget {...restProps} key={index} schema={colSchema} />)
-			)}
+			{schema.title && (<legend>{schema.title}</legend>)}
```

Comment:
i think we are missing a test here

Reply:
It's part of the test that is already in place, I just forgot to update the snapshot :/

Reply:
In fact I misread the test . Added it in the test